Documentation/Home/References.rst

```diff
@@ -88,28 +88,6 @@ Reference Manuals
                -  `12.4 <https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/>`__
                -  `11.5 <https://docs.typo3.org/m/typo3/reference-tsconfig/11.5/en-us/>`__
 
-   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
```

Comment:
You are removing here the whole TypoScript card, not only the "TypoScript in 45 Minutes" link.

Reply:
This whole card is about TypoScript in 45 Minutes as far as I can tell?

Reply:
Nope, it is only linked in the text:
![image](https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/assets/2566282/ad3d8399-fa05-46fd-b946-7a2a5db58a10)
The "12.4" and "11.5" texts link to the TS reference.

Reply:
woops, good catch!

Reply:
reverted